ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-BL1A.cfg

```diff
@@ -67,6 +70,9 @@ vrf instance MGMT
 !
 vrf instance Tenant_A_L3_VRF_Zone
 !
+vrf instance Tenant_A_OP_Zone
```

Comment:
would have been nice to not spread this VRF further, since we affect the coverage of other combinations (cases where a VRF is _not_ there)